mmv1/products/backupdr/BackupPlan.yaml

```diff
@@ -0,0 +1,209 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'BackupPlan'
+base_url: projects/{{project}}/locations/{{location}}/backupPlans
+create_url: projects/{{project}}/locations/{{location}}/backupPlans/?backup_plan_id={{backup_plan_id}}
+self_link: projects/{{project}}/locations/{{location}}/backupPlans/{{backup_plan_id}}
+immutable: true
+delete_url: projects/{{project}}/locations/{{location}}/backupPlans/{{backup_plan_id}}
```

Comment:
You should be able to remove this line, I think this is the default value used (`self_link`)